routers/member/purchase.js

```diff
@@ -307,7 +307,8 @@ router.post('/edit/:purchase_id', auth.verifyRank(ranks.pr_whitelist), async (re
             tax: xss(toDollarAmount(req.body.tax, 0)),
             submitted_by: safeString(req.auth.info.email),
             approval: 0,
-            draft: Boolean(safeString(req.body.draft))
+            draft: Boolean(safeString(req.body.draft)),
+            edited_after_rejection: req.body.approval === 1 || req.body.approval === 3
```

Comment:
I don't think the approval part comes from the request (and in any case, it wouldn't be safe to trust that part of the request since it shouldn't be controlled by the user), this would probably need to be a separate database query (query database for the purchase by finding based on `purchase_id`, then get `.approval` from that).